packages/app-project/src/screens/ProjectHomePage/components/MessageFromResearcher/locales/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "MessageFromResearcher": {
+    "noMessage": "The researcher hasn't added a message.",
```

Comment:
Isn't the default behaviour to show nothing if there's no message?

Reply:
I don't mind either way, but I can see the masonry-style design looking lopsided if we start taking out blocks. One for @beckyrother?

Reply:
If we're always going to display it, then it should probably be added to the beta checklist too, so that projects don't go live with the default message.

Reply:
Yes, it should always be there, so should be added to the beta checklist. 

If they've not added a message yet, let's say something like "Connect with the research team on Talk to learn more about this project." and take out the researcher name.